pkg/kp/kv_test.go

```diff
@@ -1,8 +1,6 @@
 package kp
 
-import (
-	"testing"
-)
+import "testing"
```

Comment:
nit: Did an automatic tool make this change? I'd rather you keep the old formatting if possible.